releases-istio/apps/elasticsearch-secret.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
```

Comment:
Move secrets to releases-istio/secrets/ please

Reply:
Moved